Example/Views/JS/Author.cshtml

```diff
@@ -9,6 +9,7 @@
 <script>
     document.getElementById('code').innerHTML = library.json.prettyPrint(@Html.Raw(ViewBag.JSON));
 </script>
+<script src="//authorapi.learnosity.com?v0.10"></script>
```

Comment:
@gonzalozawa as explained in the ticket, we do not want to render our API inside the example as if our API's changes break the API we have to go back to fix the example. Please remove this <script>